builtin/bisect--helper.c

```diff
@@ -604,48 +746,323 @@ static int bisect_start(struct bisect_terms *terms, int no_checkout,
 	string_list_clear(&states, 0);
 	strbuf_release(&start_head);
 	strbuf_release(&bisect_names);
-	return retval;
+	if (res)
```

Comment:
This variable is not declared (only `retval`, maybe?). And that is why [the build is failing](https://dev.azure.com/gitgitgadget/git/_build/results?buildId=1692).

You probably want to replace any `int retval` declaration in `bisect_start()` by `int res`.